packages/ingest/granule.js

```diff
@@ -766,12 +770,26 @@ async function moveGranuleFiles(sourceFiles, destinations) {
         });
       });
     }
+
+    let fileBucket;
+    let fileKey;
+    if (file.bucket && file.filepath) {
```

Comment:
This same `if, else` block is used a couple times throughout your changes. I'm wondering if this might be a good block to move into a separate function. Then you'd have something like `const { fileBucket, fileKey } = myCoolFunctionTM(file);`.

Reply:
We're doing the same check in a few different places, but taking different actions.  In some cases we're returning bucket / key, and others we're returning the S3 URL.  I'm hoping that, soon, we'll be able to just take these conditionals entirely out.  That'll have to wait until after we change the format of messages that are sent to the `sftracker` SNS topic, though.